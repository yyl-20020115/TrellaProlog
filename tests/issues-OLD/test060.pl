:-initialization(main).

fill(0, []).
fill(Len, [L|T]) :-
    succ(L, Len),
    !,
    fill(L, T).

main :-
    findall(p(A),
        (   between(1, 10, I),
            fill(I, A)
        ),
        C
    ),
    writeq(C), nl.
