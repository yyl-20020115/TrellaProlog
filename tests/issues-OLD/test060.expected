[p([0]),p([1,0]),p([2,1,0]),p([3,2,1,0]),p([4,3,2,1,0]),p([5,4,3,2,1,0]),p([6,5,4,3,2,1,0]),p([7,6,5,4,3,2,1,0]),p([8,7,6,5,4,3,2,1,0]),p([9,8,7,6,5,4,3,2,1,0])]
